src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/ReadyToRunReader.cs

```diff
@@ -625,8 +647,8 @@ private unsafe void EnsureHeader()
                     _pointerSize = 8;
                     break;
 
-                case (Machine) 0x6264: /* LoongArch64 */
-                    _architecture = (Architecture) 6; /* LoongArch64 */
+                case (Machine)0x6264: /* LoongArch64 */
```

Comment:
It looks like <code>Machine</code> already defines <code>LoongArch64 = 25188</code>, could we reuse that rather than open-coding the constant?